### PR TITLE
[Feature-3418][scheduler] Support task group selection

### LINKED ISSUE #3418: [Feature][scheduler] Support task group selection from DolphinScheduler
### Search before asking

- [X] I had searched in the [issues](https://github.com/DataLinkDC/dinky/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

1. The form of sending a task to DolphinScheduler will be added a new selection field -- task groups of DS, and after sending the taskGroupId value will write into DS's database. The task groups is showed relatived to current project code.

image
3. Fix a bug: when the form window is reopened after sending to DS, some field values are not correctly shown even though the request's callback contains the correct value.
fcae52dda3ea8c6b5297a78553a9212

### Use case

_No response_

### Related issues

_No response_

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by Yangxiaotian**
## Brief change log

  - *The form of sending a task to DolphinScheduler will be added a new selection field -- task groups of DS, and after sending the taskGroupId value will write into DS's database. The task groups is showed relatived to current project code.l*
  - *Fix a bug: when the form window is reopened after sending to DS, some field values are not correctly shown even though the request's callback contains the correct value.*

## Verify this pull request

This pull request is already passed by manual tests.

### CODE REVIEW

File: dinky-scheduler/src/main/java/org/dinky/scheduler/model/TaskGroup.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ */
+
+package org.dinky.scheduler.model;
+
+import java.util.Date;
+
+import lombok.Data;
+
+@Data
+public class TaskGroup {
```
Comment (Zzm0809) [inline dinky-scheduler/src/main/java/org/dinky/scheduler/model/TaskGroup.java:27]: move to dinky-scheduler module

File: dinky-scheduler/src/main/java/org/dinky/scheduler/result/DsPageData.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ */
+
+package org.dinky.scheduler.result;
+
+import java.util.List;
+
+import lombok.Data;
+
+@Data
+public class DsPageData<T> {
```
Comment (Zzm0809) [inline dinky-scheduler/src/main/java/org/dinky/scheduler/result/DsPageData.java:27]: use dinky-scheduler module's define

File: dinky-scheduler/src/main/java/org/dinky/scheduler/result/DsPageInfo.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ *  limitations under the License.
+ *
+ */
+
+package org.dinky.scheduler.result;
+
+import lombok.Data;
+
+@Data
+public class DsPageInfo<T> {
```
Comment (Zzm0809) [inline dinky-scheduler/src/main/java/org/dinky/scheduler/result/DsPageInfo.java:25]: use dinky-scheduler module's define

File: dinky-scheduler/src/main/java/org/dinky/scheduler/model/TaskGroup.java (hunk 1 above)
Comment (Yangxiaotian) [reply dinky-scheduler/src/main/java/org/dinky/scheduler/model/TaskGroup.java:27]: It's in dinky-scheduler module. Which path should this file move to ?

Comment (Zzm0809) [reply dinky-scheduler/src/main/java/org/dinky/scheduler/model/TaskGroup.java:27]: > It's in dinky-scheduler module. Which path should this file move to ?

dinky-scheduler/src/main/java/org/dinky/scheduler/model

Comment (Zzm0809) [reply dinky-scheduler/src/main/java/org/dinky/scheduler/model/TaskGroup.java:27]: > It's in dinky-scheduler module. Which path should this file move to ?

I got it wrong, sorry, no need to change here

File: dinky-scheduler/src/main/java/org/dinky/scheduler/result/DsPageInfo.java (hunk 3 above)
Comment (Zzm0809) [inline dinky-scheduler/src/main/java/org/dinky/scheduler/result/DsPageInfo.java]: > use dinky-scheduler module's define

dinky-scheduler/src/main/java/org/dinky/scheduler/result/PageInfo.java

**Review by Zzm0809 — APPROVED** [review]

**Pull request merged**